pkg/controller/volume/attachdetach/cache/actual_state_of_world_test.go

```diff
@@ -542,26 +543,33 @@ func Test_SetVolumeMountedByNode_Positive_UnsetWithoutInitialSet(t *testing.T) {
 	}
 
 	// Act
-	setVolumeMountedErr := asw.SetVolumeMountedByNode(generatedVolumeName, nodeName, false /* mounted */, false /* force unmount */)
+	setVolumeMountedErr1 := asw.SetVolumeMountedByNode(generatedVolumeName, nodeName, true /* mounted */)
```

Comment:
By default, it is true (mounted), don't need to set to true first. Maybe we could do the following
1. asw.AddVolumeNode
2. verifyAttachedVolume true (expectedMountedByNode)
3. SetVolumeMountedByNode to false
4.  verifyAttachedVolume false